koku/masu/database/azure_report_db_accessor.py

```diff
@@ -153,3 +154,36 @@ def populate_markup_cost(self, markup, bill_ids=None):
             else:
                 AzureCostEntryLineItemDailySummary.objects.\
                     update(markup_cost=(F('pretax_cost') * markup))
+
+    # pylint: disable=too-many-arguments,arguments-differ
```

Comment:
Let's try avoid these disables, if it's reasonably feasible to do so. 

I'd like to at least avoid disabling the `arguments-differ` check. I suspect that might be as easy as adding the `conflict_columns` parameter to the base class, then updating the method signatures of the other implementing classes.